### DISCUSSION TITLE
I was approved the student benefits but NO E-mail sent to me.

### DISCUSSION THREAD

**Discussion by 1191008459** [community/community#66116] [GitHub Education]
### Select Topic Area

Question

### Body

I was informed that my benefits would be available soon, but it has been an extended period, and I have yet to receive any updates.

The lack of communication and delay in the approval process has caused inconvenience and has left me in a state of uncertainty regarding the status of my account and associated benefits. I kindly request immediate attention to resolve this issue.

**Comment by advayc** [comment]
You should get a email along with the github PRO badge on your profile 72 hours after your application. During peak times it can be up to a week so just be patient and youll get the benefits soon.
